sdk/communication/communication-common/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 3.0.0 (Unreleased)
```

Comment:
Hi @DominikMe @AikoBB 

Should I leave this as "unreleased" in the PR?

Should I change the package.json version to be "3.0.0" as part of this PR ?

Reply:
Yes, after getting approval for API changes you will create a release PR where "unreleased" will be replaced by the release date. You can either change the version in package.json in this PR or as a part of the release PR